SAMv1.tex

```diff
@@ -229,6 +229,8 @@ \subsection{The header section}
   \multicolumn{2}{|l}{\tt @RG} & Read group. Unordered multiple {\tt @RG} lines are allowed.\\\cline{2-3}
   & {\tt ID}* & Read group identifier. Each {\tt @RG} line must have a unique {\tt ID}. The value of {\tt ID}
   is used in the RG tags of alignment records. Must be unique among all read groups in header section.  Read group IDs may be modified when merging SAM files in order to handle collisions.\\\cline{2-3}
+  & {\tt BC} & Barcode sequence identifying the sample/library.
```

Comment:
Should be made clear that this barcode is the one actively placed during library preparation, rather than the measured value.

Reply:
Done.